### PR TITLE
feat: added ca-cert support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                     |

### CODE REVIEW

**Review by asafMasa — APPROVED** [review]

File: helm/templates/deployment.yaml [hunk 1]
```diff
@@ -71,6 +76,10 @@ spec:
           image: {{ $cloudProviderDockerRegistryUrl }}{{ .repository }}:{{ .tag }}
           {{- end }}
           env:
+            {{- if .Values.rasterCommon.ca.secretName }}
+            - name: SSL_CERT_DIR
+              value: /usr/local/share/ca-certificates/
```
Comment (shimoncohen) [inline helm/templates/deployment.yaml:81]: Make this configureable like the mount below.

File: helm/templates/configmap.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {{- $chartName := include "gpkg-merger.name" . -}}
 {{- $tracingUrl := include "gpkg-merger.tracingUrl" . -}}
 {{- $metricsUrl := include "gpkg-merger.metricsUrl" . -}}
+{{- $httpProtocol := ternary "https://" "http://" .Values.rasterCommon.ca.secretName -}}
```
Comment (shimoncohen) [inline helm/templates/configmap.yaml:4]: Create and use instead `.Values.rasterCommon.storage.s3.ssl.enabled` or something like that.
This should be specific for S3.

**Review by shimoncohen — CHANGES_REQUESTED** [review]

File: helm/templates/configmap.yaml (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/configmap.yaml]: fixed

File: helm/templates/deployment.yaml (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/deployment.yaml]: fixed

Comment (CL-SHLOMIKONCHA) [reply helm/templates/deployment.yaml]: fixed

File: helm/templates/configmap.yaml (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/configmap.yaml]: fixed

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**